bigbluebutton-html5/imports/ui/components/join-handler/guest-wait/component.tsx

```
@@ -185,9 +185,12 @@ const GuestWait: React.FC<GuestWaitProps> = (props) => {
             <Styled.Bounce />
           </Styled.Spinner>
         )}
-        <p aria-live="polite" data-test="guestMessage">
-          {message}
-        </p>
+        <p
+          aria-live="polite"
+          data-test="guestMessage"
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: message }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)